app/src/main/java/protect/card_locker/AboutActivity.kt

```diff
@@ -2,119 +2,167 @@ package protect.card_locker
-            donate.tag = "https://catima.app/donate"
-
-            // Hide Google Play rate button if not on Google Play
-            rate.isVisible = BuildConfig.showRateOnGooglePlay
-            // Hide donate button on Google Play (Google Play doesn't allow donation links)
-            donate.isVisible = BuildConfig.showDonate
-        }
-
-        bindClickListeners()
-    }
 
-    override fun onOptionsItemSelected(item: MenuItem): Boolean {
-        return when (item.itemId) {
-            android.R.id.home -> {
-                finish()
-                true
+        setContent {
+            MaterialTheme {
+                Scaffold(
+                    topBar = {
+                        TopAppBar(
+                            title = {
+                                Text(title.toString())
+                            },
+                            navigationIcon = {
+                                IconButton(onClick = { onBackPressedDispatcher.onBackPressed() }) {
+                                    Icon(
+                                        imageVector = Icons.AutoMirrored.Filled.ArrowBack,
+                                        contentDescription = getString(R.string.back)
+                                    )
+                                }
+                            },
+                        )
+                    },
+                ) { innerPadding ->
+                    Column(modifier = Modifier.padding(innerPadding)) {
+                        CatimaAboutSection(
+                            getString(R.string.version_history),
+                            content.versionHistory,
+                            {
+                                showHTML(
+                                    getString(R.string.version_history),
+                                    content.historyInfo,
+                                    this as ComponentActivity,
+                                    "https://catima.app/changelog/"
+                                )
+                            }
+                        )
+                        CatimaAboutSection(
+                            getString(R.string.credits),
+                            content.copyrightShort,
+                            {
+                                showHTML(
+                                    getString(R.string.credits),
+                                    content.contributorInfo,
+                                    this as ComponentActivity,
+                                    null
+                                )
+                            }
+                        )
+                        CatimaAboutSection(
+                            getString(R.string.help_translate_this_app),
+                            getString(R.string.translate_platform),
+                            {
+                                OpenWebLinkHandler().openBrowser(
+                                    this as ComponentActivity?,
```

Comment:
You can obtain context in a composable using [CompositionLocal](https://developer.android.com/develop/ui/compose/compositionlocal) APIs. Check for the `LocalContext.current` method that should be available in a composable.